instance.go

```diff
@@ -92,6 +102,21 @@ func init() {
 	instanceFlags.groupId.required = true
 	cmdInstanceFake.Flags.StringVar(&instanceFlags.version, "version", "0.0.0", "Version to report.")
 	cmdInstanceFake.Flags.BoolVar(&instanceFlags.forceUpdate, "force-update", false, "Force updates regardless of rate limiting")
+
+	cmdInstanceBench.Flags.BoolVar(&instanceFlags.verbose, "verbose", false, "Print out the request bodies")
+	cmdInstanceBench.Flags.IntVar(&instanceFlags.clientsPerApp, "clients-per-app", 20, "Number of concurrent clients.")
+	cmdInstanceBench.Flags.IntVar(&instanceFlags.minSleep, "min-sleep", 1, "Minimum time between update checks.")
+	cmdInstanceBench.Flags.IntVar(&instanceFlags.maxSleep, "max-sleep", 10, "Maximum time between update checks.")
+	cmdInstanceBench.Flags.IntVar(&instanceFlags.errorRate, "errorrate", 1, "Chance of error (0-100)%.")
+	cmdInstanceBench.Flags.StringVar(&instanceFlags.OEM, "oem", "fakeclient", "oem to report")
+	cmdInstanceBench.Flags.IntVar(&instanceFlags.pingOnly, "ping-only", 0, "halt update and just send ping requests this many times.")
+	cmdInstanceBench.Flags.Var(&instanceFlags.appId, "app-id", "Application ID to update.")
+	instanceFlags.appId.required = true
+	cmdInstanceBench.Flags.Var(&instanceFlags.groupId, "group-id", "Group ID to update.")
+	instanceFlags.groupId.required = true
+	cmdInstanceBench.Flags.StringVar(&instanceFlags.version, "version", "0.0.0", "Version to report.")
+	cmdInstanceBench.Flags.BoolVar(&instanceFlags.forceUpdate, "force-update", false, "Force updates regardless of rate limiting")
+	cmdInstanceBench.Flags.Int64Var(&instanceFlags.benchSeconds, "seconds", 10, "Benchmark duration")
```

Comment:
Please remove any flags that aren't used.

Reply:
Good catch, done